testing/framework/TestCmdTests.py

```diff
@@ -424,10 +430,13 @@ class diff_TestCase(TestCmdTestCase):
     def test_diff_re(self):
         """Test diff_re()"""
         result = TestCmd.diff_re(["abcde"], ["abcde"])
+        result = list(result)
```

Comment:
Why is this necessary? Doesn't diff_re() still return a list?

Reply:
yes.  I have thoughts of diff_re and simple_diff becoming generators like the difflib functions they shadow - it's kind of odd to have to handle the returns of some diff functions one way and some another way.  Don't know if I'll ever push that, though.